main.py

```diff
@@ -352,6 +354,29 @@ def missing_page(campaign_group=None):
                            user_info=user_info,
                            short_names=short_names)
 
+@app.route('/missing_update', methods=['POST'])
+def missing_update():
+    """
+    Endpoint to update interested notes for missing samples of the twiki database
+    """
+    data = json.loads(request.data)
+    update_time = int(time.time())
+    logging.info(data)
+    #Updating notes for missing samples
+    missing_dataset_name = data['dataset_name']
+    missing_campaign = data['campaign']
+    twiki_conn = sqlite3.connect('twiki.db')
+    twiki_cursor = twiki_conn.cursor()
+    if 'missing_nts' in data:
+        notes = data['missing_nts'].strip()
+        logging.info(notes)
+        twiki_cursor.execute('''UPDATE twiki_samples
```

Comment:
Can you please tidy-up indentation and formatting here a bit? pylint is also complaining about line 375 being too long.